src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -667,4 +667,37 @@ namespace winrt::TerminalApp::implementation
         actionArgs.Handled(true);
     }
 
+    // Method Description:
+    // - Raise a IdentifyWindowsRequested event. This will bubble up to the
+    //   AppLogic, to the AppHost, to the Peasant, to the Monarch, then get
+    //   distributed down to _all_ the Peasants, as to display info about the
+    //   window in _every_ Peasant window.
+    // - This action is also buggy right now, because TeachingTips behave
+    //   weird in XAML Islands. See microsoft-ui-xaml#4382
+    // Arguments:
+    // - <unused>
+    // Return Value:
+    // - <none>
+    void TerminalPage::_HandleIdentifyWindows(const IInspectable& /*sender*/,
+                                              const ActionEventArgs& args)
+    {
+        _IdentifyWindowsRequestedHandlers(*this, nullptr);
+        args.Handled(true);
+    }
+
+    // Method Description:
+    // - Display the "Toast" with the name and ID of this window.
+    // - Unlike _HandleIdentifyWindow**s**, this event just displays the window
+    //   ID and name in the current window. It does not involve any bubbling
+    //   up/down the page/logic/host/manager/peasant/monarch.
+    // Arguments:
+    // - <unused>
+    // Return Value:
+    // - <none>
+    void TerminalPage::_HandleIdentifyWindow(const IInspectable& /*sender*/,
+                                             const ActionEventArgs& args)
+    {
+        IdentifyWindow();
+        args.Handled(true);
+    }

```

Comment:
I hate to be this guy, but should we combine both actions and make this an arg? Then we could just pass it via `ActionEventArgs`? So like...

```js
{ "command": { "action": "identifyWindow", "scope": "current" } },
{ "command": { "action": "identifyWindow", "scope": "all" } },

// we could also add scope: desktop, but like, I don't see a reason for that tbh.
```

Reply:
So I don't care about this one too much - I _really_ can't imagine a third context that we'd ever use for `identifyWindow`, so maybe this is just fine as is?

Reply:
I really don't care. If folks _actually bind these_ i'll be _shocked_.